tools/sign-installer.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Documentation:
+# - https://sectigo.com/faqs/detail/EV-Code-Signing-Certificates-Collection/kA01N000000brbF
+# - https://www.reddit.com/r/PowerShell/comments/kj88m2/exporting_code_signing_certificates_from_an_exe/
+# - https://blog.codeinside.eu/2017/11/30/signing-with-signtool-dont-forget-the-timestamp/
+# - https://docs.microsoft.com/en-us/archive/blogs/ieinternals/everything-you-need-to-know-about-authenticode-code-signing
+#
+# This requires the SafeNet Client Application installed and running on the CI
+# box. Note that the USB token with the EV certificate is not visible when
+# logged in via RDP, only when you are logged in via AnyDesk/Chrome Desktop or
+# locally. Be sure to enable single logon in SafeNet so that you only have to
+# enter the password once.
```

Comment:
@t-b mention that changes to this setting require a reboot?

Reply:
@timjarsky Good point. I also revised the instructions to be better worded.